app/src/main/res/layout/movie_details_activity.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent">
+
+    <FrameLayout
+        android:id="@+id/content_fragment"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
```

Comment:
Sure, sounds good, let's leave it as `FrameLayout`.

But yeah, the general idea (not critical for this training exercise) is to keep the view hierarchy as flat as possible. Consider that above your activity/fragment layouts there are also additional layers that Android add (you can see it if you use `uiautomatorviewer`).